components/constellation/constellation.rs

```diff
@@ -1165,7 +1165,7 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             }
             FromScriptMsg::GetChildBrowsingContextId(browsing_context_id, index, sender) => {
                 // We increment here because the 0th element is the parent browsing context itself
-                let result = self.all_descendant_browsing_contexts_iter(browsing_context_id)
+                let result = self.fully_active_descendant_browsing_contexts_iter(browsing_context_id)
                     .nth(index + 1)
```

Comment:
This seems odd, if nothing else our naming is out of sync. Is this intended to return the i'th child browsing context or the i'th descendant?

Reply:
The ith child, because grandchildren and their offspring does not need to be included.

Reply:
Oh, I see what's wrong with the naming. I'll double check all the use cases and see if they're indeed all simply looking at the children only and not their descendants as well.